pkg/controller/directvolumemigrationprogress/directvolumemigrationprogress_controller.go

```diff
@@ -276,6 +284,13 @@ func (r *ReconcileDirectVolumeMigrationProgress) reportContainerStatus(pvProgres
 		exitCode := containerStatus.LastTerminationState.Terminated.ExitCode
 		pvProgress.Status.ExitCode = &exitCode
 		pvProgress.Status.ContainerElapsedTime = &metav1.Duration{Duration: containerStatus.LastTerminationState.Terminated.FinishedAt.Sub(containerStatus.LastTerminationState.Terminated.StartedAt.Time).Round(time.Second)}
+	case !containerStatus.Ready && pod.Status.Phase == kapi.PodPending && containerStatus.State.Waiting != nil && containerStatus.State.Waiting.Reason == ContainerCreating:
+		// if pod is not in running state after 10 mins of its creation, raise a
+		if time.Now().Sub(pod.CreationTimestamp.Time) > TimeLimit {
```

Comment:
NIT: do we want `time.Now().UTC().Sub(pod.CreationTimestamp.Time.UTC())` instead of `time.Now().Sub(pod.CreationTimestamp.Time)` just to be sure the timezone is common?

Reply:
+1